src/data/contributors/RitwikMehta.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Ritwik Mehta",
+  "skills": ["C++", "Flutter/Dart", "FullStack"],
+  "cause": "I code to build. To be able to teach one day.",
```

Comment:
"I code for" is already there in the string so you only have to write the cause you code for.